pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -52,40 +47,14 @@ type Client interface {
 	) (*drapbv1.NodeUnprepareResourceResponse, error)
 }
 
-// Strongly typed address.
-type draAddr string
-
 // draPluginClient encapsulates all dra plugin methods.
 type draPluginClient struct {
-	pluginName          string
-	addr                draAddr
-	nodeV1ClientCreator nodeV1ClientCreator
+	pluginName string
+	plugin     *Plugin
 }
 
 var _ Client = &draPluginClient{}
 
-type nodeV1ClientCreator func(addr draAddr) (
-	nodeClient drapbv1.NodeClient,
-	closer io.Closer,
-	err error,
-)
-
-// newV1NodeClient creates a new NodeClient with the internally used gRPC
-// connection set up. It also returns a closer which must be called to close
-// the gRPC connection when the NodeClient is not used anymore.
-// This is the default implementation for the nodeV1ClientCreator, used in
-// newDRAPluginClient.
-func newV1NodeClient(addr draAddr) (nodeClient drapbv1.NodeClient, closer io.Closer, err error) {
-	var conn *grpc.ClientConn
-
-	conn, err = newGrpcConn(addr)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	return drapbv1.NewNodeClient(conn), conn, nil
-}
-
```

Comment:
I believe this was pulled from code related to CSI and never really exercised in DRA. I am fine removing it rather than adapting it work with the new abstractions being introduced as part of this PR.